src/modules/roc_packet/target_stdio/roc_packet/address_to_str.cpp

```diff
@@ -19,27 +21,36 @@ address_to_str::address_to_str(const Address& addr) {
 
     switch (addr.version()) {
     case 4: {
-        if (!addr.get_ip(buffer_, sizeof(buffer_))) {
-            roc_log(LogError, "address to str: can't format ip");
-        }
-
-        const size_t blen = strlen(buffer_);
-        if (snprintf(buffer_ + blen, sizeof(buffer_) - blen, ":%d", addr.port()) < 0) {
-            roc_log(LogError, "address to str: can't format port");
+        if (addr.has_multicast_iface()) {
+            if (snprintf(buffer_, sizeof(buffer_), "%s@%s:%d", ip_to_str(addr).c_str(),
+                         multicast_iface_to_str(addr).c_str(), addr.port())
```

Comment:
Since we didn't implement this notation, I suggest to use something more explicit like:

    <ip>:<port> miface <ip>

Reply:
Done